app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -644,10 +648,42 @@ fun AppPresentationView(app: App) {
           overflow = TextOverflow.Ellipsis,
         )
       }
+      AppRating(
+        modifier = Modifier
+          .width(4.dp),
+        rating = app.pRating
+      )
     }
   }
 }
 
+@Composable
+fun AppRating(
+  modifier: Modifier = Modifier,
```

Comment:
The spacing between the icon and text is always the same and the usages all use the same modifier, so it doesn't seem necessary to have this parameter